uo_sys_framework/Core/cfgXEH/init/ClientPostInit.sqf

```diff
@@ -35,5 +35,7 @@ if (hasInterface) then {
     LOG("HC Post Init");
     if (EGETMVAR(AI,ViewDistance_Enforce,false)) then {
         setViewDistance EGETMVAR(AI,ViewDistance,2500);
+    } else {
```

Comment:
I think `ViewDistance` and `AIViewDistance` should both be Core vars. Since these macros are all used in the Core component they can be internal (non E prefixed) macros.

Reply:
I changed Core to Vision. Should I change it back?

Reply:
These are all core related variables, so yes return it to Core.

Reply:
I would say Core as they are unique enough to be identified just by `ViewDistance` in the vars, but any grouping of vars that go together can be moved into their own component, just have to move every instance of it other to that new component. It's just a matter of how many vars are going to be in that component and how similar they are with other var names.